python/tank_vendor/shotgun_api3/shotgun.py

```diff
@@ -224,10 +224,10 @@ def __init__(self, host, meta):
         except AttributeError:
             self.version = None
         if not self.version:
-            raise ShotgunError("The ShotGrid Server didn't respond with a version number. "
```

Comment:
You can revert the changes in python/tank_vendor/shotgun_api3. This is a copy of python-api that will be made when we release a new version.